packages/metal-clay-button/src/MetalClayButton.js

```diff
@@ -61,11 +61,10 @@ MetalClayButton.STATE = {
 	 * @instance
 	 * @memberof MetalClayButton
 	 * @type {?string}
-	 * @default 'default'
+	 * @default undefined
 	 */
 	style: {
 		validator: validators.string,
-		value: 'default'
```

Comment:
Not sure this is an issue but looks odd to have the type declared as string but it can return undefined. I'd say we either update the  jsdoc or make the default value an empty string